NOTICE

```diff
@@ -162,7 +162,7 @@ Please visit the Netty web site for more information:
 
   * http://netty.io/
 
-Copyright 2011 The Netty Project
+Copyright 2014 The Netty Project
```

Comment:
The license changes are overdue update of the Netty 4.x license from 3.x's version